src/amo/components/HomeHeroBanner/index.js

```diff
@@ -15,7 +16,6 @@ import './styles.scss';
 
 type InternalProps = {|
   ...WithExperimentInjectedProps,
-  _tracking: typeof tracking,
```

Comment:
can remove the tracking import above to make linting happy :) ?

Reply:
d'oh!